plugins/email/test/email.spec.js

```diff
@@ -578,7 +580,12 @@ test('sends an email via a template with ICU content', async (t) => {
   t.true(html.includes('une réduction de 20 %'))
   t.true(html.includes('Votre colis sera livré à votre domicile'))
   t.true(html.includes('à partir du 15/03/2019 (vendredi 15 mars 2019)'))
-  t.true(html.includes('13:45:12'))
+
+  // get hours, minutes and seconds from Date object
+  // because they may differ depending on local time
+  const availableDate = new Date(availableDateStr)
+  const time = `${availableDate.getHours()}:${availableDate.getMinutes()}:${availableDate.getSeconds()}`
```

Comment:
According to MDN, [`getX` date methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getHours) depend on local time so how are users supposed to do to not depend on local time of our infrastructure? This is what we should test.

Reply:
Thanks, I'll seek methods to allow users to be specify dates independently from our infrastructure.

Reply:
Unless we clearly state in our docs + API reference that our servers use UTC time.

Reply:
I found a way to specify the timezone in email content and dates will be formatted according to it without any change in ICU messages (no custom format).

I use [`intl-messageformat` custom formatters](https://github.com/formatjs/formatjs/tree/master/packages/intl-messageformat#formatters) and create a custom behaviour if timezone if specified.

https://github.com/stelace/stelace/blob/b40d6f55f4199022469f56ca8f67c2b5fbe617cc/plugins/email/util/i18n.js#L79-L106

If timezone isn't specified, UTC time will be used.

Reply:
Before coming up with this custom formatter solution, I tried the `timeZone` parameter from `Intl.DateTimeFormat`. However our Intl polyfill doesn't support it.
https://github.com/andyearnshaw/Intl.js/blob/4bd382c5df206f328b35ca23579904f603b84f0b/src/12.datetimeformat.js#L198

The date timezone formatter extends the library `intl-messageformat` which is already used by email plugin.

That's right, we should mention in the docs that dates are rendered with UTC time by default.
I think the `timezone` parameter is coherent with what Stelace API offers in Availability and Task API to help users handling timezone issues.